compiler/rustc_lint/src/levels.rs

```diff
@@ -1015,7 +1015,12 @@ impl<'s, P: LintLevelsProvider> LintLevelsBuilder<'s, P> {
                     |lint| {
                         lint.set_arg("name", lint_id.lint.name_lower());
                         lint.note(fluent::lint_note);
-                        add_feature_diagnostics(lint, &self.sess.parse_sess, feature);
+                        add_feature_diagnostics(
```

Comment:
```suggestion
                        add_feature_diagnostics_for_issue(
```
Could you directly use `add_feature_diagnostics_for_issue` here, and avoid adding the `feature_from_cli` parameter to `add_feature_diagnostics`/`create_feature_err`?